### DISCUSSION TITLE
Mason.vim upgrade

### DISCUSSION THREAD

**Discussion by Ryanlier** [LunarVim/LunarVim#4351] [Q&A]
Thank you for this wonderful integrated nvim distro.  Yesterday, I ran checkhealth and found out that the Mason.vim used was 1.0 (now the latest is 1.7) . I'm wondering how to upgrade the Mason.vim to the latest version. I couldn't find it in the doc
 Thank you 😀

**Comment by StudentWeis** [comment]
I have the same problem
